core/lib/generators/refinery/cms/cms_generator.rb

```diff
@@ -112,6 +125,37 @@ def create_decorators!
       end
     end
 
+    def create_heroku_procfile!
+      create_file "Procfile" do
+        "web: bundle exec puma -C config/puma.rb"
+      end unless destination_path.join('Procfile').file?
+
+      create_file "config/puma.rb" do
+%{threads Integer(ENV['MIN_THREADS']  || 1), Integer(ENV['MAX_THREADS'] || 16)
+
+workers Integer(ENV['PUMA_WORKERS'] || 3)
+
+rackup DefaultRackup
+port ENV['PORT'] || 3000
+environment ENV['RACK_ENV'] || 'development'
+preload_app!
+
+on_worker_boot do
+  # worker specific setup
+  ActiveSupport.on_load(:active_record) do
+    ActiveRecord::Base.establish_connection
+  end
+
+  # If you are using Redis but not Resque, change this
+  if defined?(Resque)
+    Resque.redis = ENV["OPENREDIS_URL"] || "redis://127.0.0.1:6379"
```

Comment:
I think we can omit this. There's no benefit using Resque or Redis for clean RefineryCMS.